compiler/p/codegen/PPCInstruction.cpp

```diff
@@ -593,33 +622,76 @@ bool TR::PPCSrc2Instruction::usesRegister(TR::Register *reg)
 
 void TR::PPCSrc2Instruction::assignRegisters(TR_RegisterKinds kindToBeAssigned)
    {
-   TR::Register           *virtualRegister1 = getSource2Register();
-   TR::Register           *virtualRegister2 = getSource1Register();
+   TR::Register           *virtualRegister1 = getSource1Register();
+   TR::Register           *virtualRegister2 = getSource2Register();
    TR::RealRegister       *assignedRegister;
    TR::Machine *machine = cg()->machine();
+   bool excludeGPR0 = getOpCodeValue() == TR::InstOpCode::stxvl;
+
```

Comment:
Same undesirable thing about excluding gr0.  Plus, gr0 interference is not done in this case it seemed?

Reply:
I think the interference you're referring to is being handled in the constructor for `PPCSrc2Instruction`. Or are you referring to something else?

Reply:
missed that sneaky code.